File: problems/27/Solution.swift
class Solution {
    func removeElement(_ nums: inout [Int], _ val: Int) -> Int {
        var length = 0
        for i in 0..<nums.count {
            if nums[i] != val {
                nums[length] = nums[i]
                length += 1
            }
        }
        
        return length
    }
}

File: problems/27/Solution.java
package tw.idv.rchu.leetcode;

public class Solution {
    public int removeElement(int[] nums, int val) {
        int length = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                nums[length] = nums[i];
                length++;
            }
        }

        return length;
    }
}